dapps/marketplace/src/pages/notifications/NotificationRow.js

```diff
@@ -14,8 +13,22 @@ const Row = ({ node, identity, onClick }) => {
   const title = <b>{get(node, 'offer.listing.title')}</b>
   const event = get(node, 'event.event')
   let description = `${name} ${event} ${title}`
-  const nameLink = <Link to={`/user/${get(node, 'party.id')}`}>{name}</Link>
+  const buyerId = get(node, 'offer.buyer.id')
+  const partyId = get(node, 'party.id')
+  const nameLink = <Link to={`/user/${partyId}`}>{name}</Link>
+
+  const partyRole =
+    partyId === buyerId
+      ? fbt('Buyer:', 'notifications.buyer')
+      : fbt('Seller:', 'notifications.seller')
 
+  const shortenWallet = wallet => {
```

Comment:
Can use `utils/formatHash` here. Need to document all these util functions :)

Reply:
👍